src/http.cc

```diff
@@ -88,7 +89,7 @@ HttpStateData::HttpStateData(FwdState *theFwdState) :
     surrogateNoStore = false;
     serverConnection = fwd->serverConnection();
 
-    if (fwd->serverConnection() != NULL)
+    if (fwd->serverConnection())
```

Comment:
Please avoid out of scope changes.

Reply:
I recall we had agreed to modernize as we go over close changes, do I mis rembember?

Reply:
> I recall we had agreed to modernize as we go over close changes, do I mis rembember?

We agreed to modernize code that has to be modified for other reasons: Imagine that you postpone all modernization decisions until after everything else in the PR is settled. At that time, you should (in most cases) modernize the lines that the diff shows as added or modified. This rule of thumb is meant to reduce arguments about modernization scope, simplify associated decisions, and reduce diff noise.

In this particular case, this line will not be in the everything-but-modernization diff described above and, hence, should not be modernized.